core/src/main/java/org/frankframework/management/bus/endpoints/Monitoring.java

```diff
@@ -111,28 +112,28 @@ public Message<String> addMonitorOrTrigger(Message<?> message) {
 		MonitorManager mm = getMonitorManager(configurationName);
 
 		Monitor monitor;
-		ITrigger trigger;
 		if(name != null) {
 			monitor = getMonitor(mm, name);
-			trigger = SpringUtils.createBean(mm.getApplicationContext(), Trigger.class);
-			updateTrigger(trigger, message);
-			monitor.addTrigger(trigger);
+			ITrigger trigger = SpringUtils.createBean(mm.getApplicationContext(), Trigger.class);
+			try {
+				updateTrigger(trigger, message);
+				monitor.addTrigger(trigger);
```

Comment:
I would like to invoke validation in or after the `updateTrigger` but then there's no validation possible that monitor is set in the trigger.

Perhaps that is Okay though, and we move that check into `configure()` only and out of `validate()`.

So I'm not entirely sure yet about the change but mostly this is the way I think it should go.